pkg/protocol/sybilprotection/sybilprotectionv1/sybilprotection.go

```diff
@@ -311,6 +315,52 @@ func (o *SybilProtection) EligibleValidators(epoch iotago.EpochIndex) (accounts.
 	return validators, nil
 }
 
+// IsActive returns true if the given validator is currently active.
+func (o *SybilProtection) IsActive(validatorID iotago.AccountID, epoch iotago.EpochIndex) bool {
+	activeCandidates := o.performanceTracker.EligibleValidatorCandidates(epoch)
+	return activeCandidates.Has(validatorID)
+}
+
+// OrderedRegisteredValidatorsList returns the currently known list of registered validator candidates for the given epoch.
+func (o *SybilProtection) OrderedRegisteredValidatorsList(epoch iotago.EpochIndex) ([]*apimodels.ValidatorResponse, error) {
+	candidates := o.performanceTracker.ValidatorCandidates(epoch)
+	activeCandidates := o.performanceTracker.EligibleValidatorCandidates(epoch)
+
+	validatorResp := make([]*apimodels.ValidatorResponse, 0, candidates.Size())
+	if err := candidates.ForEach(func(candidate iotago.AccountID) error {
+		accountData, exists, err := o.ledger.Account(candidate, o.lastCommittedSlot)
+		if err != nil {
+			return err
+		}
+		if !exists {
+			return ierrors.Errorf("account of committee candidate does not exist: %s", candidate)
+		}
+		if accountData.StakeEndEpoch < epoch {
```

Comment:
Is the StakeEndEpoch inclusive or exclusive? in other words: is the StakeEndEpoch the last epoch in which your stake is active or is the first epoch such stake doesn't apply anymore?

Reply:
From TIP: Feature whose `End Epoch` is the current one or has passed, it is no longer considered for validator selection and the. So the cond was incorrect, changed to `<=`